src/main/java/at/hannibal2/skyhanni/config/features/garden/laneswitch/LaneswitchNotificationSettings.java

```diff
@@ -0,0 +1,48 @@
+package at.hannibal2.skyhanni.config.features.garden.laneswitch;
+
+import at.hannibal2.skyhanni.utils.LorenzColor;
+import com.google.gson.annotations.Expose;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorDropdown;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorSlider;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorText;
+import io.github.moulberry.moulconfig.annotations.ConfigOption;
+
+public class LaneswitchNotificationSettings {
+
+    @Expose
+    @ConfigOption(name = "Notification Text", desc = "The Text to be displayed as the Notification.")
+    @ConfigEditorText
+    public String notificationText = "Lane Switch incoming.";
+
+    @ConfigOption(name = "Notification Text Color", desc = "Notification Text Color. §eIf Chroma is gray, enable Chroma in Chroma settings.")
+    @Expose
+    @ConfigEditorDropdown
+    public LorenzColor notificationColor = LorenzColor.YELLOW;
+
+    @Expose
+    @ConfigOption(name = "Notification Duration", desc = "The time the Notification is displayed.")
+    @ConfigEditorSlider(
+        minValue = 1F,
+        maxValue = 5F,
```

Comment:
I feel like the maximum here should be brought in line with the maximally earliest notification so that the notification lasts until the end of the lane.